docs/source/extend_kedro/index.md

```diff
@@ -6,4 +6,5 @@
 common_use_cases
 plugins
 architecture_overview
+../starters/create_a_starter
```

Comment:
There's another thing (not for this PR) which is that our folder structure doesn't reflect the TOC structure. This might or might not be a good thing - looks like the folders group pages by topics, and the TOC by use case. Wondering if we could unify those for our own sanity.

Reply:
As you say. Not for this PR. We should wait to make further changes (obviously this is a change, but unavoidable) when we have agreed IA improvements.